1.9/development/index.md

```diff
@@ -1,97 +1,103 @@
-* The `postInstallNotes` parameter gives the user information they'll need after the installation. Focus on providing a documentation URL, a tutorial, or both. For example: `"postInstallNotes": "Thank you for installing the Unicorn service.\n\n\tDocumentation: http://<service-url>\n\tIssues: https://github.com/"`
-* The `postUninstallNotes` parameter gives the user information they'll need after an uninstall. For example, further cleanup before reinstalling again and a link to the details. A common issue is cleaning up ZooKeeper entries. For example: `postUninstallNotes": "The Unicorn DC/OS Service has been uninstalled and will no longer run.\nPlease follow the instructions at http://<service-URL> to clean up any persisted state" }`
-
- [1]: http://mesosphere.github.io/universe/
- [2]: http://mesosphere.github.io/universe/#publish-a-package-1
- [3]: /docs/1.9/administration/securing-your-cluster/
- [4]: https://github.com/mesosphere/universe
- [5]: http://mesosphere.github.io/universe/#submit-your-package
- [6]: https://github.com/mesosphere/dcos-helloworld
+This section describes the developer-specific DC/OS components, explaining what is necessary to package and provide your own service on DC/OS.
+
+The Mesosphere Datacenter Operating System (DC/OS) provides the optimal user experience possible for orchestrating and managing a datacenter. If you are an Apache Mesos developer, you are already familiar with developing a framework. DC/OS extends Apache Mesos by including a web interface for health checks and monitoring, a command-line, a service packaging description, and a [repository](/docs/1.9/usage/repo/) that catalogs those packages.
+
+# <a name="universe"></a>Package Repositories
+
+The DC/OS Universe contains all of the services that are installable DC/OS. For more information on DC/OS Universe, see the [GitHub Universe repository(https://github.com/mesosphere/universe).
+
+All services in the package repositories are required to meet a certain standard as defined by Mesosphere. For details on submitting a DC/OS service, see [Contributing a package](http://mesosphere.github.io/universe/#publish-a-package-1).
+
+# <a name="adminrouter"></a>Admin Router and web interface integration
+
+By default, a DC/OS service is deployed on a [private agent node](/1.9/overview/concepts/#private-agent-node). To allow configuration control or monitoring of a service by a user, the admin router proxies calls on the master node to the service in a private node on the cluster. The HTTP service endpoint requires relative paths for artifacts and resources. The service endpoint can provide a web interface, a RESTful endpoint, or both. When creating a DC/OS CLI subcommand it is common to have a RESTful endpoint to communicate with the scheduler service.
```

Comment:
This link is broken: `[private agent node](/1.9/overview/concepts/#private-agent-node)`.